src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -718,7 +718,7 @@ public function fromFloat($value)
     }
 
     /**
-     * Return a decimal as string
+span class="pl-s1">      * Return a decimal as string.
```

Comment:
Hah, I just experienced this bug as well with the new suggested changes feature from Github. It's still very beta :)

Reply:
:( Will do it manually :)